demo/AppNavigator.js

```diff
@@ -1,9 +1,8 @@
 import { createStackNavigator, createAppContainer } from "react-navigation";
 
 import HomeScreen from "./screens/HomeScreen";
-import ListSellOrdersScreen from "./screens/ListSellOrdersScreen";
-import SellOrderClaimScreen from "./screens/SellOrderClaimScreen";
-import SellOrderExecuteScreen from "./screens/SellOrderExecuteScreen";
+import ListLandsForSaleScreen from "./screens/ListLandsForSaleScreen";
```

Comment:
The only problem is that land can't be plural

Reply:
Hm... is `ListLandForSaleScreen` correct?

Reply:
Yeah, that works. The “list” word still shows that it’s for multiple parcels